File: Golang/RomanToInteger/RomanToInteger_test.go
package RomanToInteger

import "testing"

func TestRomanToInt(t *testing.T) {

	if romanToInt("DCXXI") == 621 {
		t.Log("pass")
	} else {
		t.Error("failed")
	}

	if romanToInt("MCMXCVI") == 1996 {
		t.Log("pass")
	} else {
		t.Error("failed")
	}

	if romanToInt("CMXCIX") == 999 {
		t.Log("pass")
	} else {
		t.Error("failed")
	}

}

File: Python/RomanToIntegerTest.py
from unittest import TestCase

from RomanToInteger import RomanToInteger


class TestRomanToInteger(TestCase):
    def test_romanToInt(self):
        rtt = RomanToInteger()
        # Expected: 621 = 500 + 100 + 10 + 10 +1
        self.assertTrue(rtt.romanToInt("DCXXI") == 621)
        # Expected: 1996 = 1000 - (100 - 1000) - (10 - 100) + 5 + 1
        self.assertTrue(rtt.romanToInt("MCMXCVI") == 1996)
        # Expected: 999 = - (100 - 1000) - (10 - 100) - (1 - 10)
        self.assertTrue(rtt.romanToInt("CMXCIX") == 999)
